ocaml/fstar-lib/generated/FStar_Monotonic_Seq.ml

```diff
@@ -158,4 +158,7 @@ let testify_seqn :
   fun i ->
     fun l ->
       fun log ->
-        fun max -> fun ctr -> let n = FStar_HyperStack_ST.op_Bang ctr in ()
\ No newline at end of file
+        fun max ->
+          fun ctr ->
+            let n = FStar_HyperStack_ST.op_Bang ctr in
+            FStar_HyperStack_ST.testify ()
```

Comment:
interestingly, this was being dropped previously (`testify: ((heap -> Type) -> ST unit)`)